tracer/src/Datadog.Trace/Configuration/IntegrationId.cs

```diff
@@ -40,6 +40,7 @@ internal enum IntegrationId
         AwsSdk,
         AwsSqs,
         AwsSns,
+        AwsLambda,
```

Comment:
Just highlighting this comment at the bottom 🙂 

```csharp
// If you add an integration here, please don't forget to impact the allow / block list of telemetry config in the telemetry repository.
```

Reply:
This is done in this PR: https://github.com/DataDog/dd-go/pull/106971